handler_test.go

```diff
@@ -466,35 +466,38 @@ var handlePanicTests = []struct {
 	expect: `bad handler function: last argument cannot be used for Unmarshal: bad route tag "httprequest:\\"BAD /foo\\"": invalid method`,
 }}
 
-func (*handlerSuite) TestHandlePanicsWithBadFunctions(c *gc.C) {
-	for i, test := range handlePanicTests {
-		c.Logf("%d: %s", i, test.expect)
-		c.Check(func() {
-			testServer.Handle(test.f)
-		}, gc.PanicMatches, test.expect)
+func TestHandlePanicsWithBadFunctions(t *testing.T) {
+	c := qt.New(t)
+
+	for _, test := range handlePanicTests {
+		c.Run(test.expect, func(c *qt.C) {
```

Comment:
using "expect" as the test name here is going to lead to some mighty awkward test names. Perhaps consider adding a testName field and using that?

Reply:
done